### ISSUE TITLE
Emails: missing alt text

### ISSUE THREAD

**Issue by aardrian** [Devographics/Monorepo#318]
The emails announcing the survey and following up on past surveys are missing alt text for the big logo.

Since that logo takes up my entire mobile screen when viewing the email, and since all that is displayed when my images don't load is the text “Insert alt text here”, it looks broken.

![Email message with subject “Announcing the first-ever State of… HTML?!” followed by a broken image icon alongside the text “Insert alt text here” with none of the rest of the message visible.](https://github.com/Devographics/Monorepo/assets/1376607/f8241fad-a81c-4626-992f-43d0c5b158b8)

**Comment by SachaG** [comment]
Whoops, thanks for catching that! Good thing I only sent that email to 100,000 people!

**Comment by eric-burel** [comment]
@SachaG I still have smth like this on the State of React email:
![image](https://github.com/Devographics/Monorepo/assets/7983005/c862d2bf-6601-4209-8237-b1fe8ecc5cc3)

**Comment by aardrian** [comment]
Still happening, this one in the State of CSS email.

![A large black box with a broken image icon and blue text that reads “Insert alt text here”.](https://github.com/user-attachments/assets/5963039a-85fe-45d6-9ea8-a07d232f400c)

This one has been open for over a year and, frankly, seems like a straightforward one to fix across all the "State of" email templates.

**Comment by SachaG** [comment]
Sorry, I typically duplicate last year's template to create a new one for this year's survey, and I must've picked one where that issue wasn't fixed. I'll double check them all now.
